File: Clase4/Tarea/tarea4.py
no_dia= int(input("Dame numero del 1-7: "))
dia_semana = {
    1: "lunes",
    2: "martes",
    3: "miercoles",
    4: "jueves",
    5: "viernes",
    6: "sabado",
    7: "domingo"   
}
print(dia_semana.get(no_dia, "No existe el día"))

File: Clase4/Tarea/tarea4.java
package Clase4.Tarea;
import java.util.Scanner;

public class tarea4 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Dame un numero del 1-7: -");
        int no_dia=sc.nextInt();
        
        switch (no_dia) {
            case 1:
                System.out.println("Lunes");
                break;
            case 2:
                System.out.println("Martes");
                break;
            case 3:
                System.out.println("Miércoles");
                break;
            default:
                System.out.println("Día inválido");
        }
       
        



    }   
}

